src/app/components/passwordToggleButton.component.ts

```diff
@@ -0,0 +1,28 @@
+import { PasswordState } from '../components';
+
+export class PasswordToggleButton{
```

Comment:
Please, add Component to the class name or move into other directory

Reply:
done